docs/concepts/flows.md

```diff
@@ -255,6 +255,8 @@ Hello Marvin!
 
 You can get a quick sense of the structure of your flow using the `.visualize()` method on your flow. Calling this method will attempt to produce a schematic diagram of your flow and tasks without actually running your flow code.
 
+!!! warning "Functions and code not inside of @flows or @tasks will still be run when calling `.visualize()`. This may have unintended consequences."
```

Comment:
```suggestion
!!! warning "Functions and code not inside of @flows or @tasks will still be run when calling `.visualize()`. This may have unintended consequences. Place your code into tasks to avoid unintended execution."
```